fbsat-cli/src/main/kotlin/ru/ifmo/fbsat/cli/command/infer/AbstractInferCommand.kt

```diff
@@ -38,6 +40,19 @@ abstract class AbstractInferCommand<AutomatonType : Any>(name: String) : CliktCo
             Globals.IS_USE_ASSUMPTIONS = isUseAssumptions
             Globals.IS_DUMP_VARS_IN_CNF = isDumpVarsInCnf
             Globals.IS_DEBUG = isDebug
+            Globals.NEGATIVE_TREE_OPTIMIZATIONS = negativeTreeOptimizations.also {
+                if (it == NegativeTreeOptimizations.OPT1 || it == NegativeTreeOptimizations.OPT2) {
+                    check(Globals.IS_FORBID_TRANSITIONS_TO_FIRST_STATE) {
+                        "You should forbid transitions to first state to use negative-tree-optimization-{1/2}"
```

Comment:
Would be nice to add to error messages the actual flags that have to be specified.